source/frontend/src/components/Table/PageSizeSelector.tsx

```diff
@@ -40,12 +39,13 @@ export const TablePageSizeSelector: React.FC<React.PropsWithChildren<IProps>> =
     <Menu options={pageSizeOptions} alignTop={alignTop}>
       <div style={{ display: 'flex' }}>
         <StyledText>Show</StyledText>
+        {/*the selector appears to capture the click event which prevents the page change from being registered*/}
         <Form.Control
           size="sm"
           value={`${selected}`}
           type="number"
-          style={{ width: 50, marginLeft: 10, marginRight: 10 }}
-          onChange={noop}
+          style={{ width: 50, marginLeft: 10, marginRight: 10, backgroundColor: 'white' }}
+          disabled
```

Comment:
setting the input here to disabled fixed the input from taking the click event from the parent menu.

Reply:
I saw this on the select ad well and updated it so that if you pass something on the onclick it will still update the form